src/tbp/monty/frameworks/models/motor_policies.py

```diff
@@ -1220,19 +1220,12 @@ def get_inverse_agent_rot(self):
         identity pose, which will be aquired by transforming the original pose by the
         inverse
 
-        NB Magnum appears to be a library used by Habitat; i.e. it is still grounded
-        in quaternions
-
         Returns:
             Inverse quaternion rotation.
         """
-        inverse_magnum_rotation = hab_utils.common.quat_to_magnum(
-            self.state["agent_id_0"]["rotation"]
-        ).inverted()
-        inverse_quaternion_rotation = hab_utils.common.quat_from_magnum(
-            inverse_magnum_rotation
-        )
-        return inverse_quaternion_rotation
+        [w, x, y, z] = qt.as_float_array(self.state[self.agent_id]["rotation"])
+        [x, y, z, w] = rot.from_quat([x, y, z, w]).inv().as_quat()
```

Comment:
Maybe add a comment here clarifying that magnum and scipy rotation use different quaternion conventions.

Reply:
yes, sorry thats what I meant